test/python/providers/test_fake_backends.py

```diff
@@ -161,3 +169,27 @@ def test_delay_circuit(self):
         qc.measure_all()
         res = transpile(qc, backend)
         self.assertIn("delay", res.count_ops())
+
+    @data(0, 1, 2, 3)
+    def test_converter(self, opt_level):
+        backend = FakeYorktown()
+        backend_v2 = BackendV2Converter(backend)
+        self.assertIsInstance(backend_v2, BackendV2)
+        res = transpile(self.circuit, backend_v2, optimization_level=opt_level)
+        job = backend_v2.run(res)
```

Comment:
Does `backend.run` require Aer?  If so, we should decorate the test with a `skipUnless`.

Reply:
For fake backends it does not, the fake backends will opportunistically use aer if it's installed but fallback to use basicaer if it's not (and also emit a runtime warning about not simulating noise because aer isn't installed)